tla-types/src/main/scala/at/forsyte/apalache/tla/typecheck/TypeCheckerTool.scala

```diff
@@ -96,7 +95,13 @@ class TypeCheckerTool(annotationStore: AnnotationStore, inferPoly: Boolean, useR
         annotations.filter(_.name == StandardAnnotations.TYPE_ALIAS).foreach { annot =>
           annot.args match {
             case Seq(AnnotationStr(text)) =>
-              aliases += parseTypeAlias(decl.name, text)
+              val (alias, assignedType) = parseTypeAlias(decl.name, text)
+              if (!ConstT1.isAliasReference(alias)) {
+                val msg =
+                  s"Operator ${decl.name}: Deprecated syntax in type alias $alias. Use camlCase of Type System 1.2."
```

Comment:
Why do we have enforce anything about the case of the letters if we are requiring a `$` to reference the alias? Shouldn't the case be irrelevant then?

Reply:
In principle, yes, but we still have the all-caps syntax for old aliases and uninterpreted types. I just wanted some disambiguation here.

Reply:
fwiw, I don't see the value in applying this unneeded constraint.